CHANGELOG.rst

```diff
@@ -10,8 +10,17 @@ This document describes changes between each past release.
 
 - Remove pypy supports. (#1049)
 
+**Internal changes**
+
+- Permission schema children fields are now set during initialization instead of on
+  deserialization (#1046).
+- Request schemas (including validation and deserialization) are now isolated by method
+  and endpoint type (#1047).
+- Move generic API schemas (e.g TimeStamps and HeaderFields) from `kinto.core.resource.schema`
+  to a sepate file on `kinto.core.schema`.
```

Comment:
Hmm, that's interesting. I agree `ResourceSchema` should be on `kinto.core.resource` (and I'm keeping it there), but I'm not so sure about other things like Timestamps or Header fields that could be reused in other parts of the core.

Most of these schemas are imported on `kinto.core.resource` so they are still accessible. Only Timestamp and URL aren't, and they aren't used anywhere on Kinto. Do you know why they are there?